### DISCUSSION TITLE
NoTimestamp trait for Adonis 5 ?

### DISCUSSION THREAD

**Discussion by LeCoupa** [adonisjs/core#1934] [Help]
How can we avoid our models to have the `createdAt` and `updatedAt` automatically created at insertion.

> ER_BAD_FIELD_ERROR: Unknown column 'created_at' in 'field list'

We used to have a NoTimestamp trait in Adonis 4 to deal with this. How can we get this behavior back in Adonis 5? :)

**Comment by thetutlage** [comment]
Simply don't define the columns?

**Reply by LeCoupa** [reply]
I have found what the issue is. I have a parent class called `SlugifyModel` with a hook inside to create a slug from the column name.

```javascript
export default class SlugifyModel extends BaseModel {
  @beforeSave()
  public static async generateSlugFromName(instance: any) {
    instance.slug = StringHelpers.slugify(instance.name)
  }
}
```

And a class called Industry that is extending `SlugifyModel`.

```javascript
export default class Industry extends SlugifyModel {
  @column({ isPrimary: true })
  public id: number

  @column()
  public name: string

  @column()
  public slug: string
}
```

Now, in my `IndustrySeeder`, whenever I try to save an industry with `create`, `createMany`, `updateOrCreate`, I get: 

> insert into `industries` (`created_at`, `name`, `slug`, `updated_at`) values ('2020-11-20 11:09:48', 'information technology and services', 'information-technology-and-services', '2020-11-20 11:09:48') - ER_BAD_FIELD_ERROR: Unknown column 'created_at' in 'field list'

Even if `created_at` and `updated_at` are not defined as a column in any model. Did I miss something?

**Reply by dawidow** [reply]
A slightly different question. What do you use, what kind of package to create slugs?

**Reply by LeCoupa** [reply]
@dawidow this one https://www.npmjs.com/package/slugify :)
